src/sched/batch.cpp

```
@@ -0,0 +1,426 @@
+    }
+
+    result.completed = true;
+    result.checked_tus = sweep.checked;
+    result.failed_tus = sweep.failed;
+    result.findings = sweep.findings;
+    if(sweep.findings != 0) {
+        result.exit_code = 1;
+    }
+    if(sweep.failed != 0) {
+        result.exit_code = 2;
+    }
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)